doc/Tex/RAM_SCB_install.tex

```diff
@@ -1,180 +1,72 @@
-is to set an environment variable that RAM-SCB will search for upon 
-installation.  They can be set with the following commands:
+RAM-SCB must be explicitly pointed to libraries installed in non-standard places. There are several ways for RAM-SCB to find the required external libaries, summarized in Table \ref{tab:libs}. The most permanant and convenient method is to set an environment variable that RAM-SCB will search for upon installation. They can be set with the following commands:
 
 Bash Shell: {\tt export NETCDFDIR=}\textit{installation path}
 
 C Shell: {\tt setenv NETCDFDIR ``}\textit{installation path}\verb*l''l
 
-Placing these commands into configuration files (e.g. {\tt \verb*l~l/.bashrc} 
-or {\tt \verb*l~l/.cshrc}) will load the environment variables at login.  This 
-is preferable to typing the commands every session.  Note that the installation
-path listed should point to the top level directory for the installation.  
-For example, {\tt /Users/ram\_user/ncarg} is sufficient but 
-{\tt /Users/ram\_user/ncarg/lib} will cause an error.
+Placing these commands into configuration files (e.g. {\tt \verb*l~l/.bashrc} or {\tt \verb*l~l/.cshrc}) will load the environment variables at login. This is preferable to typing the commands every session. Note that the installation path listed should point to the top level directory for the installation. For example, {\tt /Users/ram\_user/netcdf} is sufficient but {\tt /Users/ram\_user/netcdf/lib} will cause an error.
 
-Alternatively, it is possible to set the install paths for the external 
-libraries using Config.pl during installation.  The use of this method is 
-detailed in the next section.  Even if you have set the environment variables, 
-using the Config.pl switches will override them.  These switches are 
-\emph{only} recognized in conjunction with the {\tt -install} switch 
-of Config.pl.
+Alternatively, it is possible to set the install paths for the external libraries using Config.pl during installation. The use of this method is detailed in the next section. Even if you have set the environment variables, using the Config.pl switches will override them. These switches are \emph{only} recognized in conjunction with the {\tt -install} switch of Config.pl.
 
-Finally, if you set neither the environment variables or the Config.pl 
-switches, the Fortran compiler will look for the libaries in the default 
-directories.  This may be useful to test if the libraries are already 
-properly installed by the system administrator.  
+Finally, if you set neither the environment variables or the Config.pl switches, the Fortran compiler will look for the libaries in the default directories. This may be useful to test if the libraries are already properly installed by the system administrator.  
 
-\emph{If you change the library locations by any method, 
-you must reinstall the code.} 
+\emph{If you change the library locations by any method, you must reinstall the code.} 
```

Comment:
For those with less experience, it might be nice to point out that the library locations are found at runtime. So at runtime they need to be in the same place - which means that if you compile with a certain set of modules loaded, the configuration must be the same at runtime.

Reply:
Good point